src/image/Filter.cc

```diff
@@ -191,7 +178,7 @@ void Filter::_initRegistry() {
     _nameMap = new NameMap;
     _idMap = new IdMap;
 
-    define(FilterProperty(unknownFilter, lsst::pex::policy::Policy(), true));
+    define(FilterProperty(unknownFilter, daf::base::PropertySet(), true));
```

Comment:
Why doesn't this one get an **lsst::** lead-in?  (not saying it's wrong...I just don't know!)

Reply:
In C++ when you're inside a namespace you don't usually need to use, so the `lsst::` was never necessary even on the old one.